app/Repositories/FeedRepository.php

```diff
@@ -73,10 +73,10 @@ public function updateOrCreate(array $feed)
             $feed->slug = Feed::slugfy($feed->id, $feed->name);
             $feed->save();
 
-            dispatch(new RegisterEpisodesFeed([
+            RegisterEpisodesFeed::dispatch([
                 'id' => $feed->id,
                 'url' => $feed->url,
-            ]), true);
+            ]);
```

Comment:
tem como incluir aqui o segundo parametro (true), pois quando o feed é criado, ele força a busca de novos episodios (caso contrario, ele só irá buscar novos episodios se o headers do feed foi atualizado recentemente)

Reply:
O parâmetro `true` foi removido pois não é utilizado pela função `dispatch`.

Reply:
esse parametro, na verdade, é do construtor da RegisterEpisodesFeed class, estava errado antes, sendo passado como sendo da helper dispatch()

foi feito um PR anterior que resolvia isso, já mergeei ele e agora é só resolver os conflitos referente a isso